cdap-ui/app/features/mapreduce/mapreduce.less

```diff
@@ -27,4 +27,9 @@ body.state-mapreduce{
       border-right: 1px solid black;
     }
   }
+  .btn.btn-default.no-border{
+    &:focus{
+      outline: none;
+    }
+  }
```

Comment:
Since this is repeated several times, should we extract this to either theme.less or common.less?